Documentation/AvailableOperations.md

```diff
@@ -0,0 +1,251 @@
+- Bubbles
+- Groups
+- Comments
+- Content
+- CustomStyle
+- Activities
+- Notifications
+- User Followings
+  - Fetching users followed by user
+- Partner Applications
+- Reports
+  - User
+  - Gallery
+  - Creation
+  - Comment
+- Batch fetching
+  - Creators
+  - Managers
+  - Galleries
+
+### Initialization settings
+### Authentication
+
+#### Login
+To login, call `login` method, with username and password passed as parameters.
+```Swift
+public func login(username username: String, password: String, completion:ErrorClosure?)
+```
+
+#### Logout
+To logout, call `logout` method. It will remove all tokens stored in APIClient, also `creatubblesAPIClientUserChanged(...)` delegate method will be called.
+```Swift
+public func logout()
+```
+#### Switching users (without password)
+Switching user without password is a bit more complicated operation.
+You can fetch all users which you can login to using:
+```Swift
+public func getSwitchUsers(pagingData: PagingData?, completion: UsersClosure?) -> RequestHandler
+```
+Next, use `switchUser(...)` method to obtain OAuth token for user which would you like to login as.
+```Swift
+public func switchUser(targetUserId targetUserId: String, accessToken: String, completion: SwitchUserClosure?) -> RequestHandler
+```
+
+After this, you should have enough data to call `setSessionData(...)` method, which will switch OAUth tokens.
+```Swift
+public func setSessionData(data sessionData: SessionData)
+```
+### Landing URL's
+#### Supported landing URLs
+We're supporting fetching landing urls for:
+- About us
+- Terms of use
+- Privacy policy
+- Registration
+- User profile
+- Explore
+- Creation
+- Forgot password
+- Upload guidelines
+- Account dashboard
+
+#### Landing URL for Creation
+For Creation landing URL, please use:
+```Swift
+public func getLandingURL(creationId creationId: String, completion: LandingURLClosure?) -> RequestHandler
+```
+#### Other Landing URLs
+For other types of landing urls, use:
+```Swift
+public func getLandingURL(type type: LandingURLType?, completion: LandingURLClosure?) -> RequestHandler
+```
+
+### Creators management
+#### Fetching currently logged in user
+To fetch currently logged in user, use `getCurrentUser(...)` method:
+```Swift
+public func getCurrentUser(completion: UserClosure?) -> RequestHandler
+```
+
+#### Fetching single user by identifier
+To fetch user by his identifier, use `getUser(...)` method:
+```Swift
+public func getUser(userId userId: String, completion: UserClosure?) -> RequestHandler
+```
+
+#### Fetching single user account data
+Account data contains more detailed information about user. To fetch it, use `getUserAccountData(...)` method:
+```Swift
+public func getUserAccountData(userId userId: String, completion: UserAccountDetailsClosure?) -> RequestHandler
+```
+You can read more about account data [here](https://partners.creatubbles.com/api/#account-details)
+
+#### Fetching Creators
+To fetch creators of a user, use a `getCreators(...)` method:
```

Comment:
'To fetch creators of a user, use a `getCreators(...)` method:'
-> 'To fetch user's creators, use...'